code/modules/reagents/reagent_containers.dm

```diff
@@ -67,7 +67,8 @@
 	return
 */
 /obj/item/weapon/reagent_containers/afterattack(obj/target, mob/user , flag)
-	return
+	if(user.a_intent == "harm")
+		return ..()

```

Comment:
One of the devil's potential weaknesses is being attacked by a fresh harvest.  This allows you to attack someone with fruits/vegetables.